ciscoumbrellainvestigate_connector.py

```diff
@@ -60,7 +60,8 @@ def _make_rest_call(self, endpoint, request_params, action_result):
         status_code = None
 
         try:
-            r = requests.get(self._base_url + endpoint, headers=headers, params=request_params)
+            r = requests.get(  # nosemgrep: python.requests.best-practice.use-timeout.use-timeout
+                self._base_url + endpoint, headers=headers, params=request_params)
```

Comment:
Should we use `timeout=DEFAULT_TIMEOUT` here as well?

Reply:
hmm, I was concerned that might break something since the function this lives in is called by every action as a general request handler. but I guess I'll add it and see if the integration tests are still OK

Reply:
I feel 30seconds is a good amount to wait for. If it takes more than 30seconds, it should fail.